pom.xml

```diff
@@ -14,6 +14,16 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- for use in maven-surefire-plugin, maven-failsafe-plugin, jacoco-maven-plugin -->
+    <it.failsafe.argLine></it.failsafe.argLine>
+
+    <!-- Sonar & JaCoCo (related profile: sonar)-->
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    <sonar.organization>nuxeo</sonar.organization>
+    <sonar.projectName>Nuxeo Connect Client</sonar.projectName>
+    <it.jacoco.destFile>${session.executionRootDirectory}/target/jacoco-it.dump</it.jacoco.destFile>
```

Comment:
My IDE told me that:

> Cannot resolve symbol 'session.executionRootDirectory'

Could you explain briefly about how to configure this value? Is it a variable injected by Jenkins job?

Reply:
Plugin JaCoCo has already 2 well defined locations for storing the unit test coverage report and integration test coverage report. Could you explain the benefit of using another path? Is it because we want to merge both reports together, and store in an unique location?

```
$ mvn help:describe -Dplugin=org.jacoco:jacoco-maven-plugin -Ddetail
...
jacoco:prepare-agent
  Available parameters:
    destFile (Default: ${project.build.directory}/jacoco.exec)
      User property: jacoco.destFile
      Path to the output file for execution data.
...
jacoco:prepare-agent-integration
  Available parameters:
    destFile (Default: ${project.build.directory}/jacoco-it.exec)
      User property: jacoco.destFile
      Path to the output file for execution data.
```